src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -108,24 +134,42 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
           </div>
         )
         : (
-          <CourseInstanceTable
-            academicYear={acadYear}
-            courseList={
-              showRetired
-                ? currentCourses
-                : currentCourses.filter(
-                  ({ spring, fall }): boolean => (
-                    fall.offered !== OFFERED.RETIRED
-                        && spring.offered !== OFFERED.RETIRED)
+          <>
+            <Dropdown
+              id="select-view-dropdown"
+              name="select-view-dropdown"
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setCurrentViewId(event.currentTarget.value);
+              }}
+              labelPosition={POSITION.LEFT}
+              label="View"
+              value={currentViewId}
+              options={[
+                ...views.map((view) => ({
+                  label: view.name,
+                  value: view.id,
+                })),
+              ]}
+            />
```

Comment:
I'd suggest wrapping this `Dropdown` in the `VerticalSpace` component, as in `FacultyAdmin` and `CourseAdmin`.

Reply:
Good idea. Implemented in 275ee8d71d1b4db3f2ab6f63450f48b3b799749f